songhyun/db/migrations/20221102061711_users.sql

```diff
@@ -0,0 +1,13 @@
+-- migrate:up
+CREATE TABLE users (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(30) NOT NULL,
+    password VARCHAR(50) NOT NULL,
+    email VARCHAR(100) NOT NULL,
+    userProfileImage VARCHAR(500),
```

Comment:
```suggestion
    profile_image_url VARCHAR(500),
```
- 테이블 내 컬럼명을 작성할 때에는 `언더바(_)`를 활용하여 `snake case`로 작성해 주세요.
- 또한 모든 네이밍 컨벤션이 동일해야 합니다. (CamelCase, snake_case를 섞어서 사용하면 안됨)